tools/raspi_debug_server/README.md

```diff
@@ -0,0 +1,101 @@
+# Raspberry as server to debug/flash
+
+Features :
+ * Firmware upload
+ * Acces the USB shell                                          
+ * Connect to the CAN traffic 
+
+Raspberry:
+ * port 3333 : gdb
+ * port 4444 : telnet
+
+ To do:
+ * launch the ./shell-shh.sh script via a cvra cmd
+ * modify the script so we can launch other commande too (bootloader_*)
+
+
+# Command (from your computer) 
+Flash remotly
+```BASH
+make flash-pi
+```
+Get chibios shell
+```BASH
+./shell-shh.sh
+```
+
+# Setup rasppi
+## Update and upgrade the pi
+```BASH
+sudo apt update -y && sudo apt upgrade -y
+```
+## Install Python3 and Co
+```BASH
+sudo apt install python3
+```
+```BASH
+sudo apt install python3-pip && pip3 install cvra-bootloader
+```
+```BASH
+pip3 install pyserial
+```
+Update $PATH to find python
+```BASH
+vim ~/.bashrc 
+```
+by adding this at the eof
+```BASH
+export PATH=$PATH:~/.local/bin/
+```
+
+## Setup a service (with systemd) to launch our openocd at boot
+```BASH
+sudo mv openocd-remote.service /lib/systemd/system/.
+```
+```BASH
+sudo systemctl daemon-reload
+```
+```BASH
+sudo systemctl enable openocd-remote.service
+```
+Check if it is enabled:
+```BASH
+sudo systemctl list-unit-files | grep enabled
+```
+then
+```BASH
+sudo systemctl start openocd-remote.service
+```
+or 
+```BASH
+reboot
```

Comment:
can also be grouped in one block

```md
## Setup a service (with systemd) to launch our openocd at boot
```BASH
sudo cp openocd-remote.service /lib/systemd/system/
sudo systemctl daemon-reload
sudo systemctl enable openocd-remote.service

# check if it is enabled:
sudo systemctl list-unit-files | grep enabled

# start and reboot
sudo systemctl start openocd-remote.service
reboot
```